main.go

```diff
@@ -36,6 +36,12 @@ func main() {
 	webhookListenerPort := os.Getenv("WEBHOOK_LISTENER_PORT")
 	workflowsDefTOMLFile := os.Getenv("WORKFLOWS_DEF_TOML_FILE")
 
+	if debug {
+		log.Println("Debug:", debug)
```

Comment:
I would say, let's show value of debug irrespective of its value. I wanna revisit logging by the entire program to make it better in near future.

Reply:
Makes sense, fixed. Linking relevant issue: https://github.com/Automattic/neurobot/issues/1